docs/src/main/sphinx/admin/spill.rst

```diff
@@ -133,6 +133,6 @@ Window functions
 Window functions perform an operator over a window of rows, and return one value
 for each row. If this window of rows is large, a significant amount of memory may
 be needed. When spill to disk for window functions is enabled, if there is not enough
-memory, intermediate sorted results are written to disk. They are loaded back and
```

Comment:
technically, they are `sorted`.
`cumulated` is also not that meaningful. Remove it altogether?

Reply:
@sopel39 Did you think we should remove this sentence?

Reply:
@sopel39 Did you think we should remove this sentence?

Reply:
I mean `sorted/cumulated` word itself

Reply:
@sopel39 Ok, I remove the keyword `sorted/cumulated`, more details in https://github.com/trinodb/trino/pull/8881/commits/bbaed40b88c88a41f9a04786ef22c2f8f3cbeb50